.travis.yml

```diff
@@ -1,8 +1,7 @@
 sudo: false
 language: go
 go:
-  - 1.5
-  - 1.6
+  - 1.7
```

Comment:
I'd continue to test 1.6, if only to make sure we don't accidentally use a 1.7 feature. I believe Go always supports one version back.